.vitepress/sidebars/develop.ts

```diff
@@ -1,4 +1,4 @@
-import { Fabric } from "../types";
+import {Fabric} from "../types";
```

Comment:
Seems to be an accidental style change.
```suggestion
import { Fabric } from "../types";
```